src/streaming/controllers/BufferController.js

```diff
@@ -344,7 +350,7 @@ function BufferController(config) {
     }
```

Comment:
`triggerEvent(Events.SEEK_TARGET, {time: e.from, mediaType: type, streamId: streamInfo.id});`
I think we can remove the mediaType and the streamId here